x-pack/plugins/security_solution/public/flyout/right/components/threat_intelligence_overview.tsx

```diff
@@ -45,36 +45,43 @@ export const ThreatIntelligenceOverview: React.FC = () => {
     });
   }, [eventId, openLeftPanel, indexName]);
 
-  const { loading, threatMatchesCount, threatEnrichmentsCount } = useFetchThreatIntelligence({
+  const {
+    loading: threatIntelLoading,
+    error: threatIntelError,
+    threatMatchesCount,
+    threatEnrichmentsCount,
+  } = useFetchThreatIntelligence({
     dataFormattedForFieldBrowser,
   });
 
-  const data: InsightsSummaryPanelData[] = [
-    {
-      icon: 'image',
-      value: threatMatchesCount,
-      text: threatMatchesCount <= 1 ? THREAT_MATCH_DETECTED : THREAT_MATCHES_DETECTED,
-    },
-    {
-      icon: 'warning',
-      value: threatEnrichmentsCount,
-      text: threatMatchesCount <= 1 ? THREAT_ENRICHMENT : THREAT_ENRICHMENTS,
-    },
-  ];
-
-  const error: boolean =
-    !eventId ||
-    !dataFormattedForFieldBrowser ||
-    (threatMatchesCount === 0 && threatEnrichmentsCount === 0);
+  const error: boolean = !eventId || !dataFormattedForFieldBrowser || threatIntelError;
```

Comment:
it is more like `isVisible` correct?

Reply:
that is correct, but the way I was seeing it is, as the rendering is really done within the `InsightsSubSection` component, I wanted to consider here an error, and let the other component make the decision on the action, which is rending `null` or in other words hiding the component entirely